buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -144,10 +117,9 @@ def groupcollide(
     group2: AbstractGroup,
     dokill: bool,
     dokill2: bool,
-    collided: Optional[Callable[[Sprite, Sprite], bool]] = None,
-) -> Dict[Sprite, Sprite]: ...
+    collided: Optional[Callable[[Sprite, Sprite], bool]] = ...,
+) -> Mapping[Sprite, Sprite]: ...
```

Comment:
I think it should return Dict type here, not Mapping? Since it returns a dict.

Reply:
Fixed